docs/en/getting_started.md

```diff
@@ -19,9 +19,9 @@ Please refer to [data\_preparation.md](./dataset_preparation.md) for data prepar
 ## Body Model Preparation (Optional)
 
 If you want to obtain keypoints3d, the body model is not necessary.
-If you want to infer SMPL as well, you can prepare the body\_model as follows.
+If you want to infer SMPL as well, you can prepare the body model as follows.
 
-- [SMPL](https://smpl.is.tue.mpg.de/) v1.0 is used in our experiments.
+- [SMPL](https://smpl.is.tue.mpg.de/) v1.0.0 is used in our experiments. Please register to get access to the downloads section.
   - Neutral model can be downloaded from [SMPLify](https://smplify.is.tue.mpg.de/).
   - All body models have to be renamed in `SMPL_{GENDER}.pkl` format. <br/>
     For example, `mv basicModel_neutral_lbs_10_207_0_v1.0.0.pkl SMPL_NEUTRAL.pkl`
```

Comment:
Shall we provide the full path here to give better idea about where to find and where to put the model: `mv DOWNLOAD_PATH/smplify_public/code/models/basicModel_neutral_lbs_10_207_0_v1.0.0.pkl XRMOCAP_PATH/xrmocap_data/body_models/smpl/SMPL_NEUTRAL.pkl`?

Reply:
Here, we could explain how to rename a file, and the file path is mentioned below in the file structure.